src/erk/cli/activation.py

```diff
@@ -90,10 +95,15 @@ def render_activation_script(
 
     logging_helper = _render_logging_helper()
 
+    # Build optional post-cd commands section
+    post_cd_section = ""
+    if post_cd_commands:
+        post_cd_section = "\n# Post-cd commands\n" + "\n".join(post_cd_commands) + "\n"
+
     return f"""# {comment}
 {logging_helper}
 {cd_command}
-# Unset VIRTUAL_ENV to avoid conflicts with previous activations
+{post_cd_section}# Unset VIRTUAL_ENV to avoid conflicts with previous activations
```

Comment:
this should run after the new venv is activated

Reply:
Resolved via /erk:pr-address at 2025-12-30 06:43 AM PST - moved post_cd_commands to run after venv activation

_Addressed via `/erk:pr-address` at 2025-12-30 06:43 PST_